app/src/main/java/org/cru/godtools/fragment/GlobalDashboardFragment.kt

```diff
@@ -0,0 +1,22 @@
+package org.cru.godtools.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.viewModels
+import org.cru.godtools.databinding.FragmentGlobalDashboardBinding
+import org.cru.godtools.viewmodel.GlobalDashboardDataModel
+import java.util.Calendar
+
+class GlobalDashboardFragment : BasePlatformFragment() {
+    private var binding: FragmentGlobalDashboardBinding? = null
+    private val viewModel: GlobalDashboardDataModel by viewModels { defaultViewModelProviderFactory }
```

Comment:
you should be able to do this to simplify this a bit more
```suggestion
    private val viewModel: GlobalDashboardDataModel by viewModels()
```